packages/mask/src/plugins/NextID/components/NextIdPage.tsx

```diff
@@ -58,12 +59,15 @@ export function NextIdPage({ personaList }: NextIDPageProps) {
     const currentProfileIdentifier = useLastRecognizedIdentity()
     const visitingPersonaIdentifier = useCurrentVisitingIdentity()
     const personaConnectStatus = usePersonaConnectStatus()
-    const { reset, isVerified, action } = useNextIDConnectStatus()
+    const { reset, isVerified } = useNextIDConnectStatus()
 
     const [openBindDialog, toggleBindDialog] = useState(false)
     const [unbindAddress, setUnBindAddress] = useState<string>()
     const platform = activatedSocialNetworkUI.configuration.nextIDConfig?.platform as NextIDPlatform
     const isOwn = currentProfileIdentifier.identifier.toText() === visitingPersonaIdentifier.identifier.toText()
+    const pluginId = usePluginIDContext()
+    const isEvm = pluginId === NetworkPluginID.PLUGIN_EVM
```

Comment:
Prefer to use `enableRequirement`, spread this condition is not a good choice.

Reply:
The whole NextID plugin will be disabled.